csharp/src/Ice/UdpEndpoint.cs

```diff
@@ -200,12 +201,12 @@ internal UdpEndpoint(InputStream istr)
             {
                 MulticastInterface = argument ?? throw new FormatException(
                     $"no argument provided for --interface option in endpoint `{endpointString}'");
-
+                // TODO: should we do anything for ::0 and 0.0.0.0?
```

Comment:
Try to parse the string to check if it's a wildcard address and if that's the case don't set MulticastInterface? I think, I would do nothing like today as in the end it will result in the same as not setting "--interface".

Reply:
Should we then just alias `*` to `::0`?

Reply:
The is one difference is that we do set some socket options when MulticastInterface is set. 

https://github.com/externl/ice/blob/master/csharp/src/Ice/UdpTransceiver.cs#L699